mods/webui/src/stories/generictoggle/GenericToggle.stories.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2024 by Fonoster Inc (https://fonoster.com)
+ * http://github.com/fonoster/fonoster
+ *
+ * This file is part of Fonoster
+ *
+ * Licensed under the MIT License (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { Meta, StoryObj } from "@storybook/react";
+import { fn } from "@storybook/test";
+import { GenericToggle } from "./GenericToggle";
+
+/**
+ * This story is for the Generic Toggle component based on MUI switch component
+ * It takes a defaultvalue, value, disable and onChange.
```

Comment:
Please update the word defaultvalue as defaultValue.

Reply:
oops! will update